runtime_scan/pkg/provider/docker/helpers.go

```diff
@@ -0,0 +1,86 @@
+package docker
+
+import (
+	"fmt"
+	"github.com/openclarity/vmclarity/api/models"
+	"github.com/openclarity/vmclarity/runtime_scan/pkg/provider"
+	"os"
+	"strings"
+)
+
+func convertTags(tags map[string]string) *[]models.Tag {
+	ret := make([]models.Tag, 0, len(tags))
+	for key, val := range tags {
+		ret = append(ret, models.Tag{
+			Key:   key,
+			Value: val,
+		})
+	}
+	return &ret
+}
+
+func getScanName(config *provider.ScanJobConfig) (string, error) {
```

Comment:
`scanNameFor` is more descriptive than `getScanName` which indicates that some API call is being made

Reply:
I was trying to keep the naming consistent with what we have for other providers, e.g. `getInstanceWithID` in AWS

Reply:
func removed